x-pack/plugins/transform/public/app/sections/transform_management/transform_management_section.tsx

```diff
@@ -105,7 +105,7 @@ export const TransformManagement: FC = () => {
           <EuiText color="subdued">
             <FormattedMessage
               id="xpack.transform.transformList.transformDescription"
-              defaultMessage="Use transforms to pivot existing Elasticsearch indices into summarized or entity-centric indices."
+              defaultMessage="Use transforms to pivot existing Elasticsearch indices into summarized entity-centric indices or to find the latest document for each unique key."
```

Comment:
what about:

"... or to create an indexed view of the latest document for fast access."

Reply:
> "... or to create an indexed view of the latest document for fast access."

That sounds good, though it would be "documents" (plural), right?

Reply:
yes, "documents" :+1: